composer.json

```
@@ -3,6 +3,7 @@
     "description": "Lets you find a path to the system CA bundle, and includes a fallback to the Mozilla CA bundle.",
     "type": "library",
     "license": "MIT",
+    "minimal-dependency": "dev",
```

```suggestion
    "minimum-stability": "dev",
```

https://getcomposer.org/doc/04-schema.md#minimum-stability